### PR TITLE
[sailjail] Add long descriptions. Fixes JB#52256

### PR DESCRIPTION
**Pull request by Tomin1**
Add long descriptions to permissions and adjust short descriptions.
Change x-sailjail-transation-key to x-sailjail-transation-key-description.

### CODE REVIEW

File: permissions/Downloads.permission [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-downloads
+# x-sailjail-translation-key-description = permission-la-downloads
 # x-sailjail-description = Access downloads
```
Comment (rainemak) [inline permissions/Downloads.permission:5]: Drop "access" from the description => just "Downloads".

File: permissions/FingerprintSensor.permission [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
+# x-sailjail-translation-key-description =
 # x-sailjail-description = Access fingerprint sensor
```
Comment (rainemak) [inline permissions/FingerprintSensor.permission:5]: Fingerprint sensor

File: permissions/Mozilla.permission [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-webview
-# x-sailjail-description = Access webview
+# x-sailjail-translation-key-description = permission-la-webview
+# x-sailjail-description = Use webview
```
Comment (rainemak) [inline permissions/Mozilla.permission:5]: Maybe "Browsing web" instead of webview.

File: permissions/Documents.permission [hunk 4]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-documents
+# x-sailjail-translation-key-description = permission-la-documents
 # x-sailjail-description = Access documents
```
Comment (rainemak) [inline permissions/Documents.permission:5]: Documents

File: permissions/Microphone.permission [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog =
-# x-sailjail-translation-key =
+# x-sailjail-translation-key-description =
 # x-sailjail-description = Access microphone
```
Comment (rainemak) [inline permissions/Microphone.permission:5]: Microphone

File: permissions/Mozilla.permission (hunk 3 above)
Comment (rainemak) [reply permissions/Mozilla.permission:5]: Although in context of email "browsing web" doesn't sound good.

File: permissions/Music.permission [hunk 6]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-music
+# x-sailjail-translation-key-description = permission-la-music
 # x-sailjail-description = Access music
```
Comment (rainemak) [inline permissions/Music.permission:5]: Music

File: permissions/Pictures.permission [hunk 7]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-pictures
+# x-sailjail-translation-key-description = permission-la-pictures
 # x-sailjail-description = Access pictures
```
Comment (rainemak) [inline permissions/Pictures.permission:5]: Pictures

File: permissions/Thumbnails.permission [hunk 8]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-thumbnails
+# x-sailjail-translation-key-description = permission-la-thumbnails
 # x-sailjail-description = Access thumbnails
```
Comment (rainemak) [inline permissions/Thumbnails.permission:5]: Thumbnails

File: permissions/Videos.permission [hunk 9]
```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-videos
+# x-sailjail-translation-key-description = permission-la-videos
 # x-sailjail-description = Access videos
```
Comment (rainemak) [inline permissions/Videos.permission:5]: Videos

File: permissions/Mozilla.permission (hunk 3 above)
Comment (abranson) [reply permissions/Mozilla.permission:5]: View Web Pages maybe? or Display

Comment (rainemak) [reply permissions/Mozilla.permission:5]: Or "Web content" and long description telling about "Viewing web content". Idea is that longer description has those additional bits.

Comment (Tomin1) [reply permissions/Mozilla.permission]: Web content sounds good to me. Thanks!

**Comment by Tomin1** [comment]
There are still a few (probably unused) with word "Access". Should I change those?

**Comment by rainemak** [comment]
> There are still a few (probably unused) with word "Access". Should I change those?

Let's try to drop all of them

**Comment by Tomin1** [comment]
> > There are still a few (probably unused) with word "Access". Should I change those?
> 
> Let's try to drop all of them

Updated. None left.

**Review by rainemak — APPROVED** [review]
Much better now.

**Pull request merged**